### ISSUE TITLE
Sign up progress bar is not a list but is coded as such

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4013]
## Area or Component
Sign up flow

## Quick Summary
The progress bar at the top of the card is coded as a list, but is not semantically a list.

### How to Replicate
Inspect the progress bars underneath the "Step X of 3" header on organization sign up

## Accessibility Violation
Content is not a list but it is marked as such. https://cdc-devtools.dequecloud.com/rule-help/en/semantic-list-used-for-presentation

## Action Requested
Change the progress bar to be coded as a `<div>` or other non-semantic element instead
 
## Screenshots (optional)
![Screen Shot 2022-07-20 at 3.19.47 PM.png](https://images.zenhubusercontent.com/604699209bfd9306195da0d7/e5c98ad6-c8b4-4794-a897-ec5c4d052e39)

The "list" here is the blue/grey bars underneath "Step 1 of 3"

## Source Code
Probably the [StepIndicator](https://github.com/CDCgov/prime-simplereport/blob/a520f60ba21ab1458b3093fbdfb96e61f40101f4/frontend/src/app/commonComponents/StepIndicator.tsx#L15-L15)?

## Original Deque Issue
semantic-list-used-for-presentation | Content is not a list but it is marked as such. | Step indicator is marked up as a list but it is not a list | Content is not a list but is marked as such | https://cdc-devtools.dequecloud.com/rule-help/en/semantic-list-used-for-presentation | moderate | Yes | No | Lists | ["body > div#root:nth-of-type(1) > div.App:nth-of-type(1) > div#main-wrapper:nth-of-type(1) > div.card__background.bg-base-lightest:nth-of-type(2) > div.grid-container.maxw-mobile-lg.usa-section:nth-of-type(1) > div.card:nth-of-type(1) > div.margin-bottom-2.organization-form.usa-prose:nth-of-type(1) > div.usa-step-indicator.usa-step-indicator--counters-sm.usa-step-indicator--no-labels:nth-of-type(1) > ol.usa-step-indicator__segments.margin-top-1:nth-of-type(1)"] | Content is not a list but it is marked as such. | <ol class="usa-step-indicator__segments margin-top-1"> | wcag2a,wcag131f | rnv7@cdc.gov | Sign up - org info | https://www.simplereport.gov/app/sign-up/ |   | 2022-07-15T21:37:12.720Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
